### PR TITLE
Add example of manual resolution for sync conflicts

### PR DESCRIPTION
**Pull request by leplatrem**
~~Waiting for #8 to be merged~~

I believe that a simple demo conflict resolution could come with the boilerplate. It then becomes obvious to tweak it, and would save [some documentation reading](https://github.com/Kinto/kinto.js/pull/108).
- [x] Handle outgoing/incoming conflicts using remote records
- [x] Call `sync()` recursively until no conflict occurs

@Natim @ametaireau  r?

### CODE REVIEW

File: scripts/store.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+  sync() {
+    return this.collection.sync()
+      .then((res) => {
+        if (res.ok) {
+          return this.load();
+        }
+
+        // If conflicts, take remote version and sync again (recursively).
+        return Promise.all(res.conflicts.map(conflict => {
+          return this.collection.resolve(conflict, conflict.remote);
```
Comment (Natim) [inline scripts/store.js:43]: This is what I call a server wins policy :)

**Comment by Natim** [comment]
Should we have that in Kinto.js directly?

**Comment by almet** [comment]
I second @Natim comments here: is there any reason to not have this in Kinto.js directly?

**Comment by leplatrem** [comment]
That's why I intended to clarify the docs here https://github.com/Kinto/kinto.js/pull/108 about outgoing and incoming conflicts.

If CLIENT_WINS should also mean «_ignoring remote records on incoming conflicts_», then we should probably have this discussion there.

**Comment by Natim** [comment]
> Does CLIENT_WINS also mean «ignoring remote records on incoming conflicts»

IMHO this is the behavior I am waiting for when choosing the CLIENT-WINS behavior.

**Comment by Natim** [comment]
Actually on server side, the CLIENT_WINS is just that you don't add the `If-Match` header and the SERVER_WINS means that you provide it.

**Comment by leplatrem** [comment]
Yes I know all that. So let's discuss the behaviour of CLIENT_WINS and SERVER_WINS policies in kinto.js repo directly.

The scope of the boilerplate is to provide the basic code for developers to implement their application.
In this case, we use the default policy (MANUAL) which requires resolving conflicts.
This PR provides a small piece of code that implements a (simple) resolution and re-sync.

**Comment by Natim** [comment]
r+ then

**Pull request merged**